README.md

```diff
@@ -1,2 +1,5 @@
 # Eslint Plugin Resource Loader
-This eslint plugin contains rules specific to MediaWiki's [ResourceLoader](https://www.mediawiki.org/wiki/ResourceLoader).
\ No newline at end of file
+This eslint plugin contains rules specific to MediaWiki's [ResourceLoader](https://www.mediawiki.org/wiki/ResourceLoader).
+
+## Rules
+* `valid-package-file-require` - ensures `require`d files contain the file extension and are proper relative paths, i.e. `./foo.js` instead of `foo.js`.
```

Comment:
I would add a link to package modules page in mediawiki.org https://www.mediawiki.org/wiki/ResourceLoader/Package_modules
Finding it is hard and it contains lots of valuable information

Reply:
Good point. Done!